dart/lib/src/sentry_client.dart

```diff
@@ -97,6 +105,15 @@ class SentryClient {
           .getSentryException(event.throwable, stackTrace: event.stackTrace);
 
       event = event.copyWith(exception: sentryException);
+    } else if (_options.attachStackTrace &&
+        event.stackTrace == null &&
+        event.throwable == null &&
```

Comment:
at this point, `event.throwable` is not even used anymore, so not worth checking it

Reply:
✅